File: app-java/src/main/java/net/mEmoZz/yts/java/utilities/ToastUtil.java
package net.mEmoZz.yts.java.utilities;

import android.content.Context;
import android.widget.Toast;

/**
 * Authored by Mohamed Fathy on 08 Mar, 2018.
 * Contact: muhamed.gendy@gmail.com
 */

public class ToastUtil {

  public static void showShortToast(Context context, String msg) {
    Toast.makeText(context, msg, Toast.LENGTH_SHORT).show();
  }

  public static void showLongToast(Context context, String msg) {
    Toast.makeText(context, msg, Toast.LENGTH_LONG).show();
  }
}

File: app-kotlin/src/main/java/net/mEmoZz/yts/kotlin/utilities/ToastUtil.kt
package net.mEmoZz.yts.kotlin.utilities

import android.content.Context
import android.widget.Toast

/**
 * Authored by Mohamed Fathy on 08 Mar, 2018.
 * Contact: muhamed.gendy@gmail.com
 */

object ToastUtil {

  fun showShortToast(context: Context, msg: String?) {
    Toast.makeText(context, msg, Toast.LENGTH_SHORT).show()
  }

  fun showLongToast(context: Context, msg: String?) {
    Toast.makeText(context, msg, Toast.LENGTH_LONG).show()
  }
}
